filebeat/beater/filebeat.go

```diff
@@ -399,12 +401,12 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 	if b.Config.Output.Name() == "elasticsearch" {
 		pipelineLoaderFactory = newPipelineLoaderFactory(pipelineFactoryCtx, b.Config.Output.Config())
 	} else {
-		logp.Warn(pipelinesWarning)
+		fb.logger.Warn(pipelinesWarning)
 	}
 	moduleLoader := fileset.NewFactory(inputLoader, b.Info, pipelineLoaderFactory, config.OverwritePipelines)
-	crawler, err := newCrawler(inputLoader, moduleLoader, config.Inputs, fb.done, *once)
+	crawler, err := newCrawler(inputLoader, moduleLoader, config.Inputs, fb.done, *once, fb.logger)
 	if err != nil {
-		logp.Err("Could not init crawler: %v", err)
+		fb.logger.Errorf("could not init crawler: %v", err)
```

Comment:
I know "could ..." is more correct than "Could ...." but lets not change what gets logged with this PR, just the how the logger gets called.  Just to limit the scope of unintended consequences.   Looks like there are several instances of this.